failure-accrual/linkerd.yml

```diff
@@ -2,9 +2,17 @@ namers:
 - kind: io.l5d.fs
   rootDir: /disco
 
+telemetry:
+- kind: io.l5d.prometheus
+- kind: io.l5d.recentRequests
+  sampleRate: 0.25
+
+usage:
+  orgId: linkerd-examples-failure-accrual
```

Comment:
true they may not copy it, but wouldn't it also give us an idea of how many people ran the examples? or am I mistaken about this?